core/src/worker/activities/activity_heartbeat_manager.rs

```diff
@@ -332,7 +342,8 @@ impl ActivityHeartbeatManager {
                                 },
                             };
                         }
-                        HeartbeatExecutorAction::Report { task_token: tt, details, on_reported } => {
+                        HeartbeatExecutorAction::Report { task_token: tt, details } => {
+                            debug!("Recording hb {details:?}");
```

Comment:
This should probably be less verbose than `debug` and have a more meaningful message